mev-relay-rs/src/relay.rs

```diff
@@ -54,6 +60,12 @@ fn validate_execution_payload(
     // verify payload is valid
 
     // verify payload sends `value` to proposer
+    if execution_payload.fee_recipient() != &preferences.fee_recipient {
+        return Err(
+            Error::UnknownFeeRecipient(preferences.public_key.to_owned(), 
+            ExecutionAddress::try_from(execution_payload.fee_recipient().to_owned()).unwrap())
+        )
+    }
```

Comment:
the general pattern we are seeing is that builders will include a payment transaction but set themselves as the `fee_recipient` of the block

so as written, this validation would fail for most(all?) mainnet blocks

I'd suggest just leaving it out for now...